src/components/views/SSIContactViewItem/index.tsx

```diff
@@ -9,24 +9,29 @@ import {
   SSIContactViewItemContactUriCaptionStyled as ContactUriCaption,
   SSIContactViewItemContainerStyled as Container,
   SSIContactViewItemLogoContainerStyled as LogoContainer,
-  SSIContactViewItemPlaceholderLogoStyled as PlaceholderLogo,
   SSIContactViewItemNewStatusContainerStyled as StatusContainer,
 } from '../../../styles/components';
+import Debug, {Debugger} from 'debug';
+import {APP_ID} from '../../../@config/constants';
+import {SSILogo as Logo} from '@sphereon/ui-components.ssi-react-native';
+
+const debug: Debugger = Debug(`${APP_ID}:SSIContactViewItem`);
 
 export interface Props {
   name: string;
   uri?: string;
+  logo?: IImageAttributes;
   roles: Array<CredentialRole>;
 }
 
 const SSIContactViewItem: FC<Props> = (props: Props): JSX.Element => {
-  const {name, uri, roles} = props;
-
+  const {name, uri, roles, logo} = props;
+  debug(`logo: ${logo ? JSON.stringify(logo) : ''}`);
   return (
     <Container>
       <StatusContainer />
       <LogoContainer>
-        <PlaceholderLogo />
+        <Logo logo={logo} />
```

Comment:
I would expect the placeholder logo to still be here in case no logo is available for the issuer

Reply:
the SSILogo component which we're using has been doing the replacement by default, so there's no need to do it here